workspace/main.go

```diff
@@ -34,14 +34,41 @@ func usersHandler(w http.ResponseWriter, r *http.Request) {
 			return
 		}
 
-		services.SaveUser(newInput)
+		services.GenerateLocalOutput(newInput)
 		w.WriteHeader(http.StatusCreated)
 		return
+}
+}
+
+func blobHandler(w http.ResponseWriter, r *http.Request) {
+	switch r.Method {
+	case http.MethodGet:
+		statusCheck := status{Confirmation: "positive"}
+		statusCheckJson, err := json.Marshal(statusCheck)
+
+		if err != nil {
+			w.WriteHeader(http.StatusInternalServerError)
+			fmt.Printf("Write failed: %v", err)
+			return
+		}
+		w.Header().Set("Content-Type", "application/json")
+		w.Write(statusCheckJson)
 
+	case http.MethodPut:
+		newInput, err := services.ValidateRequestBody(r)
+		if err != nil {
+			w.WriteHeader(http.StatusBadRequest)
+			return
+		}
+
+		azureAccess := services.SetStorageAccess()
```

Comment:
We don't need to invoke `services.SetStorageAccess()` at this level. We can make it private (encapsulate) and do it inside `UploadFile` function.